tofhir-engine/src/test/scala/io/tofhir/test/FhirMappingJobManagerTest.scala

```diff
@@ -69,6 +69,11 @@ class FhirMappingJobManagerTest extends AsyncFlatSpec with BeforeAndAfterAll wit
     sourceContext = Map("source" -> FileSystemSource(path = "patients-extra.csv"))
   )
 
+  val patientTsvFileMappingTask: FhirMappingTask = FhirMappingTask(
+    mappingRef = "https://aiccelerate.eu/fhir/mappings/patient-mapping",
+    sourceContext = Map("source" -> FileSystemSource(path = "patients.tsv", options = Map("sep" -> "\\t")))
```

Comment:
We need to pass **"sep" -> "\\t"** option to parse a tsv file right ? Maybe, this option can be automatically added in FileDataSourceReader for tsv files, WDYT ?

Reply:
I'm not sure. Although the file itself is a tsv file, different character could be used as a separator. So \t seperator is not mandatory. Just as \t can be used in csv files, comma can be used in tsv files.

Reply:
I think that if you are using another separator then you should change the file format accordingly like CSV (comma separated) or SSV (semicolon-separated). In terms of user experience and consistency, we should handle it automatically.

Reply:
What if the user has no permission to change it. Is it better to simply give them an option to use whatever seperator they want?

Reply:
As a user, if I want to read from a tsv file, then I would not want to specify a separator option because it should be tab by definition. This is a really bad user experience in my opinion.

Moroever, the user can always provide another separator to override the default one.

Reply:
I agree with @dogukan10. Will you be updating accoringly @YemreGurses ?

Reply:
It's done, thanks @YemreGurses. Closing the thread.